VirtoCommerce.PricingModule.Web/Scripts/blades/item/item-prices.tpl.html

```diff
@@ -61,6 +63,20 @@
         </div>
     </div>
 </script>
+<!-- Date Cell Read -->
```

Comment:
I think using a date-time picker control to enter dates would make better the UX of that solution.